### DISCUSSION TITLE
2007/12/12/%E6%B1%BD%E8%BD%A6%E9%A6%96%E4%BF%9D

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#194] [Announcements]
# 2007/12/12/%E6%B1%BD%E8%BD%A6%E9%A6%96%E4%BF%9D

到今天为止(这是12月8号写的:)，我的小车开了两个月又10天，2200公里。星期六嘛，不用上班，于是去首保。

https://blog.qizhen.xyz/2007/12/12/%E6%B1%BD%E8%BD%A6%E9%A6%96%E4%BF%9D

**Comment by ruanqizhen** [comment]
之前博客下的留言：

小雨Louis
2009年09月1日 12:15 编辑
你那个轮胎的问题应该是 刹车绑的问题，我之前也有同样问题，直接更换了一个刹车绑就好了。

回复
竹贤
2007年12月14日 17:45 编辑
上汽车网站，请教讨论。

回复
